sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -3079,7 +3208,11 @@ SUBROUTINE MDLFLD
 	  IF(abs(vis(i,j))>24135.1)print*,'bad visbility'     &
        , i,j,Q1D(i,j),QW1(i,j),QR1(i,j),QI1(i,j)                 &
        , QS1(i,j),T1D(i,j),P1D(i,j),vis(i,j)
-	  GRID1(I,J)=VIS(I,J)
+          IF(Q1D(I,J)<spval.and.T1D(I,J)<spval.and.P1D(I,J)<spval)THEN
```

Comment:
@JesseMeng-NOAA 
Like one of previous comments, this kind of check should be placed in CALVIS
Then you can just check vis itself here
if(vis(i,j)<spval)then

Reply:
like the same reason this way we can plug in a new version of CALVIS, using the same current screening of Q, T, P.